python/IECore/ClassLoader.py

```diff
@@ -215,7 +215,7 @@ def defaultLoader( cls, envVar ) :
 			else :
 				msg( Msg.Level.Warning, "ClassLoader.defaultLoader", "Environment variable %s not set." % envVar )
 
-			loader = cls( SearchPath( os.path.expandvars( sp ) ) )
+			loader = cls( SearchPath( os.path.expandvars( sp ), os.pathsep ) )
```

Comment:
What I recommend here is to remove the separator entirely, because in a previous PR John and I discussed this and decided to do this internally in SearchPath without needing the separator on the end. Here is what I mean: https://github.com/ImageEngine/cortex/blob/master/include/IECore/SearchPath.h#L58

Reply:
Ahh, I think I get it now. I had seen that thread but took it to conclude with using separate declarations for the few files affected. I'll adjust as you outline. The idea is to push the path separators back to the Python UI code instead, correct?

Reply:
@ericmehl I think just doing ```SearchPath( os.path.expandvars( sp ) )``` should suffice, the C++ code internally will use the right separator internally, so the Python UI code shouldn't need to worry about any separator passed as an argument.